values.tf

```diff
@@ -1,23 +1,29 @@
+data "aws_region" "current" {}
+
 locals {
-  k8s_irsa_role_create             = var.enabled && var.k8s_rbac_create && var.k8s_service_account_create && var.k8s_irsa_role_create
-  k8s_service_account_name         = "${var.helm_chart_name}-${var.helm_release_name}"
   cloudwatch_group_name_containers = try(aws_cloudwatch_log_group.cloudwatch_containers[0].name, "")
   cloudwatch_group_name_nodes      = try(aws_cloudwatch_log_group.cloudwatch_nodes[0].name, "")
 
   values_default = yamlencode({
-    "podValuesChecksumAnnotation" : "true",
+    "image" : {
+      "tag" : "0.22.2-debian"
+    }
+    "role" : "Agent"
+    "service" : {
+      "enabled" : false
+    }
     "tolerations" : [{
-      "operator" : "Exists",
+      "key" : "node-role.kubernetes.io/master",
       "effect" : "NoSchedule"
     }],
```

Comment:
original tolerations setting was meant to schedule vector on every node no matter what taints are set, this new default value breaks this scheduling. Why we want to change this behaviour ?

Reply:
This is my fault, corrected